lib/content-services/src/lib/tag/tag-node-list.component.spec.ts

```diff
@@ -121,5 +122,66 @@ describe('TagNodeList', () => {
             const deleteButton: any = element.querySelector('#tag_chips_delete_test1');
             expect(deleteButton).not.toBeNull();
         });
+
+        it('should not render view more button by default', async () => {
+            component.ngOnChanges();
```

Comment:
I think this one is running automatically during the "detecChanges" call below

Reply:
It is but then without calling it tags are not loaded